testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/AdminOnlyModeTestCase.java

```diff
@@ -138,6 +142,50 @@ public void testAdminOnlyMode() throws Exception {
         domainSlaveLifecycleUtil.awaitServers(System.currentTimeMillis());
     }
 
+    @Test
+    public void testServersCannotStartInAdminOnlyMode() throws Exception {
+        final DomainClient masterClient = domainMasterLifecycleUtil.getDomainClient();
+
+        // restart master HC in admin only mode
+        ModelNode op = new ModelNode();
+        op.get(OP_ADDR).add(HOST, "master");
+        op.get(OP).set("reload");
+        op.get("admin-only").set(true);
+        domainMasterLifecycleUtil.executeAwaitConnectionClosed(op);
+
+        // Try to reconnect to the hc
+        domainMasterLifecycleUtil.connect();
+        domainMasterLifecycleUtil.awaitHostController(System.currentTimeMillis());
+
+        op = new ModelNode();
+        op.get(OP).set(READ_ATTRIBUTE_OPERATION);
+        op.get(OP_ADDR).add(HOST, "master");
+        op.get(NAME).set("running-mode");
+
+        // Validate that we are started in the --admin-only mode
+        ModelNode result = executeForResult(masterClient, op);
+        Assert.assertEquals("ADMIN_ONLY", result.asString());
+
+        result = executeOperation(masterClient, null, START_SERVERS);
+        Assert.assertTrue(result.asString(), result.get(FAILURE_DESCRIPTION).asString().contains("WFLYHC0048"));
+
+        result = executeOperation(masterClient, PathAddress.pathAddress(SERVER_GROUP, "main-server-group"), START_SERVERS);
+        Assert.assertTrue(result.asString(), result.get(FAILURE_DESCRIPTION).asString().contains("WFLYHC0048"));
+
+        result = executeOperation(masterClient, PathAddress.pathAddress(mainOne), START_SERVERS);
```

Comment:
I've just realized this test should not pass, this must be `start` instead of `start-servers`. I have to take a look what I'm doing wrong.